main.cpp

```diff
@@ -107,10 +107,26 @@ struct CarWash
     you should be able to deduce the return type of those functions based on their usage in Person::run()
     You'll need to insert the Person struct from the video in the space below.
  */
+    /*
+    void Person::run(int howFast, bool startWithLeftFoot)
+    {
+        if(startWithLeftFoot == true)
+        {
+            leftFoot.stepForward();
+            rightFoot.stepForward();
+        }
+        else
+        {
+            rightFoot.stepForward();
+            leftFoot.stepForward();
+        }
+        distanceTraveled += leftFoot.stepSize() + rightFoot.stepSize();
+    }
+    */
```

Comment:
Make this code executable.
Uncomment it and move it after the `Person` struct declaration
- [ ] revise